docker/.env.testnet.example

```
@@ -110,6 +110,7 @@ INDEXER_EXCHANGE_STATS_JOB_INTERVAL: "@every 8h"
 
 # Indexer Web3GW env #
 INDEXER_WEB3GW_HTTP_HEALTH_CHECK_ENDPOINT="http://sentry-sidecar:8888/health/readiness"
+INDEXER_ALLOW_FUNDING_NON_EXISTING_ACCOUNTS="true"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)